src/app/forms/custom-sections/psv-brakes/psv-brakes.component.ts

```diff
@@ -67,13 +63,13 @@ export class PsvBrakesComponent implements OnInit, OnChanges, OnDestroy {
           event.techRecord_brakes_dataTrBrakeThree = selectedBrake.parking;
         }
 
-        if (event?.axles) {
-          event.axles = (event.axles as Axle[]).filter(axle => !!axle?.axleNumber);
+        if (event?.techRecord_axles) {
```

Comment:
have we tested this? Why was this left as just axle?

Reply:
no clue